vendor_blueprints/refresh_datasource.py

```diff
@@ -0,0 +1,87 @@
+import tableauserverclient as TSC
+import argparse
+import sys
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_UNKNOWN_ERROR = 3
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_RESOURCE = 201
+EXIT_CODE_FINAL_STATUS_ERRORED = 211
+EXIT_CODE_FINAL_STATUS_CANCELLED = 212
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--username', dest='username', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--site_id', dest='site_id', required=True)
+    parser.add_argument('--content_url', dest='content_url', required=True)
+    parser.add_argument('--datasource_id', dest='datasource_id', required=True)
+    args = parser.parse_args()
+    return args
```

Comment:
Arguments should all be `--separated-with-a-dash` but interpreted as a variable of `--separated_with_an_underscore`